src/Microsoft.DotNet.Cli.Utils/Constants.cs

```diff
@@ -29,7 +29,7 @@ public static class Constants
         public static readonly string LibCoreClrName = LibCoreClrFileName + DynamicLibSuffix;
 
         public static readonly string RuntimeIdentifier = CurrentPlatform == Platform.Windows ? "win7-x64" :
-                                                          CurrentPlatform == Platform.Darwin ? "osx.10.10-x64" : "ubuntu.14.04-x64";
+                                                          CurrentPlatform == Platform.Darwin ? "osx.10.10-x64" : "ubuntu.{RuntimeEnvironment.OperatingSystemVersion}-x64";
```

Comment:
It looks like this constant is dead code.  If it's not, I can't understand what it would mean and how the default value of ubuntu.XX.XX-x64 could be correct.